doc/architecture/layers.sql

```diff
@@ -0,0 +1,112 @@
+-- User table
+CREATE TABLE users (
+    ...
+);
+
+-- Project table
+CREATE TABLE map (
+    ...
+);
+
+-- Custom geometry type
+CREATE DOMAIN geom_type AS GEOMETRY
+CHECK (
+    GeometryType(VALUE) IN ('POINT', 'LINESTRING', 'POLYGON') AND
+    ST_SRID(VALUE) = 4326
+);
+
+/*
+    Still up for discussion whether we want to also use cartesian coordinates
+
+    CREATE DOMAIN geom_cartesian_type AS GEOMETRY
+    CHECK (
+        GeometryType(VALUE) IN ('POINT', 'LINESTRING', 'POLYGON') AND
+        ST_SRID(VALUE) = 3857
+    );
+ */
+
+-- Base Layer table
+CREATE TABLE base_layers (
+    id SERIAL PRIMARY KEY,
+    map_id INTEGER REFERENCES map (id),
+    geometry geom_type,
```

Comment:
What is the purpose of this geometry column?

From my point of view, the only thing that needs to be stored for the base layers is a reference to the base image (in nextcloud or similar), as well as a conversion factor from image coordinates to real-world coordinates and the amount of rotation needed to align the image with geographic north.

Reply:
We probably need even 6 georeference parameters, see https://postgis.net/docs/RT_ST_SetGeoReference.html and additional parameters for scaling&rotating each alternative image.